zip-0244.rst

```diff
@@ -370,6 +372,35 @@ Reference implementation
 
 - https://github.com/zcash/librustzcash/pull/319/files
 
+============
+Alternatives
+============
+
+The zkproof components of Sapling spends and outputs could reasonably be
+construed as authorizing data, rather that information that describes 
+value transfer. As such, it was suggested that these proof values should
+not be committed to by in the transaction identifier, and should instead
+be used as inputs to the signature hash. Proof data is a potential source
+of transaction malleability, and as such this argument is worthy of 
+consideration.
```

Comment:
This hasn't been updated to what we decided to do (zkproofs and signatures as witness data and zkproofs excluded from signature input), right?